fuzzers/fuzzbench_text/Cargo.toml

```diff
@@ -28,6 +28,7 @@ clap = { version = "4.0", features = ["default"] }
 nix = "0.26"
 mimalloc = { version = "*", default-features = false }
 content_inspector = "0.2.4"
+#log = "0.4"
```

Comment:
is it better to have optional dependency for this?

https://doc.rust-lang.org/nightly/cargo/reference/features.html#optional-dependencies

Reply:
It's just there as a reminder for people (mainly me) that want to enable logging in the future